browser-test/src/program_deep_link.test.ts

```diff
@@ -130,4 +131,28 @@ test.describe('navigating to a deep link', () => {
       }),
     ).toBeAttached()
   })
+
+  test('redirects the applicant to an error info page if the program is disabled', async ({
+    page,
+    adminPrograms,
+  }) => {
+    await enableFeatureFlag(page, 'disabled_visibility_condition_enabled')
+    await test.step(`log in as admin and publish a disabled program`, async () => {
+      await loginAsAdmin(page)
+
+      const programName = 'dis1'
+      await adminPrograms.addDisabledProgram(programName)
+
+      await adminPrograms.gotoAdminProgramsPage(/* isProgramDisabled*/ true)
+      await adminPrograms.expectDraftProgram(programName)
+      await adminPrograms.publishAllDrafts()
+      await logout(page)
+    })
+
+    await test.step(`opens the deep link of the disabled program and gets redirected to an error info page `, async () => {
```

Comment:
```suggestion
    await test.step(`opens the deep link of the disabled program and gets redirected to an error info page`, async () => {
```
(blocking) I don't think it's currently enabled yet, but one of the eslint playwright rules will fail with the extra space at the end.